README.md

```diff
@@ -48,13 +48,13 @@
 
 [JSON:API](http://jsonapi.org) specification
 [reached 1.0 on 29th May 2015](http://www.programmableweb.com/news/new-json-api-specification-aims-to-speed-api-development/2015/06/10)
-and we also believe it is a big day for RESTful API-s as this specification can help you to make APIs more robust and future-proof. Woohoo Labs. Yin (named after Yin-Yang) was born to bring efficiency and elegance for your JSON:API servers.
+and we also believe it is a big day for RESTful APIs as this specification can help you to make APIs more robust and future-proof. Woohoo Labs' Yin (named after Yin-Yang) was born to bring efficiency and elegance to your JSON:API servers.
```

Comment:
The usage of "Woohoo Labs. Yin" is intentional (it's like "Microsoft Windows"), could you please bring it back? :)